client/src/hooks/swr-hooks/useSearch.ts

```diff
@@ -0,0 +1,10 @@
+import { SearchResponse } from '@/types/api-types';
+import useSWR from 'swr';
+
+export const useSearch = (searchPrams: string, page: number) => {
```

Comment:
Use objects to accept the parameters of any function, eliminating the need to know the order of the parameters.